src/sender_actuator.cpp

```diff
@@ -23,22 +23,39 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <mutex>
+#include <optional>
+#include <condition_variable>
+#include <map>
+#include <chrono>
+
 #include <linux/can.h>
+
 #include "canif.hpp"
+#include "scbdriver/LinearActuatorServiceResponse.h"
 #include "sender_actuator.hpp"
 
 sender_actuator::sender_actuator(ros::NodeHandle &n, canif &can)
     : sub_actuator{n.subscribe("/body_control/linear_actuator", queue_size, &sender_actuator::handle, this)},
-      sub_encoder{n.subscribe("/body_control/encoder_count", queue_size, &sender_actuator::handle_encoder, this)},
-      srv_actuator{n.advertiseService("/body_control/init_linear_actuator", &sender_actuator::handle_init, this)},
+      sub_srv_resp{n.subscribe("scbdriver/linear_actuator_service_response", queue_size, &sender_actuator::handle_srv_resp, this)},
+      srv_init{n.advertiseService("/body_control/init_linear_actuator", &sender_actuator::handle_init, this)},
+      srv_location{n.advertiseService("/body_control/linear_actuator_location", &sender_actuator::handle_location, this)},
       can{can}
 {
 }
 
-void sender_actuator::handle(const scbdriver::LinearActuatorControlArray::ConstPtr &msg) const
+void sender_actuator::handle(const scbdriver::LinearActuatorControlArray::ConstPtr &msg)
 {
-    if (in_service)
+    if (msg->actuators.size() != 3) {
+        ROS_WARN("Drop message with invalid size: %lu", msg->actuators.size());
+        return;
+    }
```

Comment:
CC @shujima @remmaTech12 @yakato-jun 

LexxTug Lでは2本だけ制御できれば良いはずです。
V7 x LexxTug Lのプロジェクトがあるまでは問題ないと思いますが、いずれどうするのか考える必要があると思います。

LexxTug L should be able to control only two of them.
Until there is a V7 x LexxTug L project, I don't think this will be a problem, but we will have to think about what to do eventually.